modules/external-api/src/main/java/org/opencastproject/external/common/ApiVersion.java

```diff
@@ -20,30 +20,63 @@
  */
 package org.opencastproject.external.common;
 
+import static org.opencastproject.external.common.ApiMediaType.VERSION_REG_EX_PATTERN;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 /** The existing versions */
 public enum ApiVersion {
 
   VERSION_UNDEFINED("v*.*.*"),
 
-  VERSION_1_0_0("v1.0.0");
+  VERSION_1_0_0("v1.0.0"),
+
+  VERSION_1_1_0("v1.1.0");
 
-  public static final ApiVersion CURRENT_VERSION = VERSION_1_0_0;
+  public static final ApiVersion CURRENT_VERSION = VERSION_1_1_0;
 
   private String versionString;
+  private int major;
+  private int minor;
+  private int micro;
```

Comment:
Please use the name "patch" instead of "micro" to be compliant to https://semver.org/

Reply:
good point, fixed